frame/election-provider-multi-phase/src/lib.rs

```diff
@@ -673,6 +662,10 @@ pub mod pallet {
 		#[pallet::constant]
 		type MaxElectableTargets: Get<SolutionTargetIndexOf<Self::MinerConfig>>;
 
+		/// The maximum number of winners that can be elected from the electable targets.
```

Comment:
I think we can give more context to this: 

e.g. This should be more than `desired_targets` return value. 

and probably more details.

Reply:
... and directly add tests for these assumptions into the integrity check.

Reply:
Since validator count is a storage value, I added this in the try_state checks in staking pallet